tests/integration/test_issue_2038.py

```diff
@@ -26,8 +26,16 @@
 def test_wheel_file_url_dep(tmpdir):
     # type: (Any) -> None
 
+    constraints = os.path.join(str(tmpdir), "constraints.txt")
+    with open(constraints, "w") as fp:
+        # The 20.22 release introduces a change that breaks resolution od poetry 1.3.2; so we pin
```

Comment:
```suggestion
        # The 20.22 release introduces a change that breaks resolution of poetry 1.3.2; so we pin
```

Typo, but not worth a CI round. Could tack it onto a future change.

Reply:
Fixed in #2129.